gabbi/case.py

```diff
@@ -482,13 +495,23 @@ def _test_data_to_string(self, data, content_type):
                 else:
                     return info
         else:
-            dumper_class = self.get_content_handler(content_type)
-            if dumper_class:
-                data = dumper_class.dumps(data)
+            if isinstance(data, dict):
+                data_new = {}
+                for k, v in data.items():
+                    if isinstance(v, six.string_types) \
+                       and all((r not in v for r in REPLACERS)):
```

Comment:
This might need to have a similar `$` prefix as is the case in `_replace_template`, just in case someone has a RESPONSE or ENVIRON word in the text.